.github/workflows/ci.yml

```diff
@@ -132,17 +128,26 @@ jobs:
           rm targets.tar
 
       - name: Import signing key
-        if: env.PGP_SECRET != '' && env.PGP_PASSPHRASE == ''
+        if: secrets.PGP_SECRET != '' && secrets.PGP_PASSPHRASE == ''
```

Comment:
Secrets can't be used in conditionals. I'm not sure if we can use env here if we only set it for this step (not sure what happens first).